src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlDataReader.cs

```diff
@@ -497,6 +499,10 @@ internal void Bind(TdsParserStateObject stateObj)
         // Fills in a schema table with meta data information.  This function should only really be called by
         // UNDONE: need a way to refresh the table with more information as more data comes online for browse info like
         // table names and key information
+#if NET6_0_OR_GREATER
```

Comment:
I think these conditionals work better as `#if !NETFRAMEWORK`. My main rationale for it is that `NET6_0_OR_GREATER` implies to me that once we drop support for net6, we can remove the conditional, but that's not actually the case since it is still needed for netframework support.